File: tests/fixtures/generic/vulnerable/taint_rce.js
// @audit-fixture
// @rule: taint_rce
// @expected: detected
const { execSync } = require("child_process");
const express = require("express");
const app = express();
app.get("/run", (req, res) => {
    const cmd = req.query.cmd;
    execSync(cmd);
});

File: tests/fixtures/generic/vulnerable/taint_rce.cs
// @audit-fixture
// @rule: taint_rce
// @expected: detected
using System.Diagnostics;
public class CommandController : Controller {
    public IActionResult Run() {
        var cmd = Request.Query["cmd"];
        Process.Start("cmd.exe", "/c " + cmd);
        return Ok();
    }
}
